core/src/main/java/org/elasticsearch/search/aggregations/metrics/scripted/ScriptedMetricAggregatorFactory.java

```diff
@@ -67,6 +67,8 @@ public Aggregator createInternal(Aggregator parent, boolean collectsFromSingleBu
             params = deepCopyParams(params, context);
         } else {
             params = new HashMap<>();
+        }
+        if (!params.containsKey("_agg")) {
```

Comment:
one minor thing, we try to be consistent and don't use `!` to negate a boolean. Yet we try to use `== false` since it's easier to read `!` is easy to miss. Would you make that change?